server/session/handler_loom.go

```diff
@@ -0,0 +1,92 @@
+package session
+
+import (
+	"fmt"
+	"github.com/df-mc/dragonfly/server/block"
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/sandertv/gophertunnel/minecraft/protocol"
+)
+
+const (
+	// loomInputSlot is the slot index of the input item in the loom table.
+	loomInputSlot = 0x09
+	// loomDyeSlot is the slot index of the dye item in the loom table.
+	loomDyeSlot = 0x0A
+	// loomPatternSlot is the slot index of the pattern item in the loom table.
+	loomPatternSlot = 0x0B
+)
+
+// handleLoomCraft handles a CraftLoomRecipe stack request action made using a loom table.
+func (h *ItemStackRequestHandler) handleLoomCraft(a *protocol.CraftLoomRecipeStackRequestAction, s *Session) error {
+	// First check if there actually is a loom opened.
```

Comment:
Maybe some more documentation? You say "First ...." but have no other comments

Reply:
Added.